system/lib/libcxxabi/include/cxxabi.h

```diff
@@ -33,16 +33,21 @@ class type_info; // forward declaration
 #endif
 }
 
+#ifdef __USING_WASM_EXCEPTIONS__
+#define NOTHROW noexcept
+#else
+#define NOTHROW throw()
```

Comment:
This line looks strange..  but probably because I don't understand this stuff well enough.

Reply:
Yeah exception specifiers are weird. think of `throw()` as `throw(void)` as opposed to `throw(type)`. There's a reason the syntax is deprecated and wasn't much used anyway.

Reply:
`throw()` means it is not supposed to throw any exceptions. It is in the process of being deprecated and `noexcept` is recommended for use in place of that. I guess the reason libcxxabi is still using it is just for compatibility with old C++. But anyway I didn't want to change that for non-wasm-EH cases that are currently working.